app/assets/stylesheets/components/_sidenav.scss

```diff
@@ -11,6 +11,8 @@ $sidenav-screen-width: 800px;
   }
 
   .usa-sidenav-list {
+    border: none;
```

Comment:
This affects all pages with a side nav. I tried it myself, and don't see any visible difference Chrome. What does this address?

Reply:
rails 1.4.4 added a top and bottom border to sidenav for narrower screen widths